.travis.yml

```diff
@@ -238,6 +238,9 @@ jobs:
             $TRAVIS_BUILD_DIR
         - make -j 2
         # run tests
+        #   Independent I/O in PHDF5
+        #   https://github.com/openPMD/openPMD-api/pull/554
+        - OPENPMD_HDF5_INDEPENDENT=ON
```

Comment:
it would be better if we add this for the specific test as JSON option #569, because otherwise we wildcard a good amount of potential issues in CI, but we do not have this in place yet.